src/App.jsx

```diff
@@ -1,11 +1,13 @@
-import React from 'react';
+import React from 'react';r
```

Comment:
I guess the 'r' at the end of line make the syntax invalid ;)

Reply:
Du coup je comprends pas, je n'ai pas ce 'r' dans mon code, c'est toi qui l'a modifié directement?

Reply:
make a git status